torchmetrics/functional/text/rouge.py

```diff
@@ -101,17 +106,27 @@ def _normalize_and_tokenize_text(text: str, stemmer: Optional[Any] = None) -> Se
             An input sentence.
         stemmer:
             Porter stemmer instance to strip word suffixes to improve matching.
+        normalizer:
+            A user's own normalizer function.
+            If this is none, `replacing any non-alpha-numeric characters with spaces` is default.
+            This function must take a `str` and return a `str`.
+        tokenizer:
+            A user's own tokenizer function. If this is none, `spliting by spaces` is default
+            This function must take a `str` and return `Sequence[str]`
```

Comment:
```suggestion
        normalizer:
            A user's own normalizer function.
            If this is `None``, replacing any non-alpha-numeric characters with space` is the default.
        tokenizer:
            A user's own tokenizer function. If this is ``None``, splitting by spaces is the default.
```
it is coming from typing